guides/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
+*   The parameters sent to `ffmpeg` for generating a video preview image are now
+    configurable under `config.active_storage.video_preview_arguments`.
 
+    *Brendon Muir*
```

Comment:
@brendon I think you changed the wrong CHANGELOG here

Reply:
I'm a rookie :)